### PR TITLE
Jetpack Forms: replace email response template with a basic one

### PR DESCRIPTION
**Pull request by donnchawp**
Revert the email response template introduced recently due to customer feedback.

This PR replaces the HTML template with a more basic one that doesn't need to be customised, and is similar to what users are used to already.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion
pcsBup-yo-p2#comment-451

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Apply PR and embed a contact form in a post.
Fill in the contact form.
The site owner should receive the response in a basic email, rather than the new HTML email with images.

### CODE REVIEW

**Review by CGastrell — DISMISSED** [review]
LGTM! Works fine on .COM

File: projects/packages/forms/src/contact-form/templates/email-response.php [hunk 1]
```diff
@@ -11,271 +11,25 @@
...
+<p>%2$s</p>
 
-													<a href="' . esc_url( Redirect::get_url( 'jetpack-ios' ) ) . '" style="color: #00AADC; text-decoration: none; width: 115px; display: inline-block; margin-bottom: 64px;"><img src="' . esc_url( plugin_dir_url( __DIR__ ) . 'images/app-store-badge.png' ) . '" class="footer-november-2022-app-image" width="115" height="33" style="line-height: 100%%; outline: none; text-decoration: none; border: 0 none; display: inline-block; height: 33px;"></a>
-												</td>
-											</tr>
-											<tr>
-												<td align="left" class="footer-november-2022-socials-section" style="border-collapse: collapse;">
-													<a href="https://twitter.com/jetpack" style="color: #00AADC; text-decoration: none; display: inline-block; padding-right: 22px;"><img aria-label="' . esc_html__( 'jetpack twitter page', 'jetpack-forms' ) . '" src="' . esc_url( plugin_dir_url( __DIR__ ) . 'images/twitter-dark.png' ) . '" width="20" height="20" style="height: auto; line-height: 100%%; outline: none; text-decoration: none; border: 0 none; display: inline-block;"></a>
+<!-- link to responses page -->
+<p><a href="%3$s">View Responses</a></p>
```
Comment (jeherve) [inline projects/packages/forms/src/contact-form/templates/email-response.php]: This wil need to be translated.

File: projects/packages/forms/src/contact-form/templates/email-response.php [hunk 2]
```diff
@@ -11,271 +11,25 @@
...
 
-													<a href="https://www.facebook.com/jetpackme" style="color: #00AADC; text-decoration: none; display: inline-block; padding-right:22px;"><img aria-label="' . esc_html__( 'jetpack facebook page', 'jetpack-forms' ) . '" src="' . esc_url( plugin_dir_url( __DIR__ ) . 'images/facebook-dark.png' ) . '" width="20" height="20" style="height: auto; line-height:100%%; outline: none; text-decoration: none; border: 0 none; display: inline-block;"></a>
+<!-- link to edit form -->
+<p><a href="%4$s">Edit</a></p>
```
Comment (jeherve) [inline projects/packages/forms/src/contact-form/templates/email-response.php]: This too.

**Review by jeherve — DISMISSED** [review]

**Review by CGastrell — APPROVED** [review]
:shipit:

**Comment by anomiex** [comment]
Cherry-picked in c721e2b4b1c4eb65d271bb1a82dc334f09739ad3

**Pull request merged**